lib/ts/BufferWriterFormat.cc

```diff
@@ -755,6 +756,160 @@ bwf_register_global(std::string_view name, BWGlobalNameSignature formatter)
   return ts::bw_fmt::BWF_GLOBAL_TABLE.emplace(name, formatter).second;
 }
 
+BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bwf_errno const &e)
```

Comment:
Also, what is bwf_errno? Is that the same as int (i.e. what errno is)?

Reply:
It's explained in the first file diff, for `buffer-writer.en.rst`.